### PR TITLE
Add Robot tabs with examples in Get Started and Advanced Usage sections

### PR DESCRIPTION
**Pull request by iulianatodoran**
In this PR:
- setup and examples for the _Get Started_ > _Write and execute first test for your app_ section
- robot tab in _Advanced usage_ >
  -  _Run tests from the command line_
  - _What is reverse port forwarding and when to use it_ > _How to setup reverse port forwarding_ + small fixes for the other languages to be consistent among each other
    - ~~for the Robot tab did not assign any value to the `device_id`  > need to clarify what will be there and if it is necessary to put it there (as it is for the other languages)~~ --> Update: applied Ru's suggestion and fixed it
      
        image

  - _Connect AltTester® Unity SDK running inside the app to AltTester® Server_
  - _Execute tests concurrently_
  - _Logging_ > _AltTester® Unity SDK logging_
  - _Logging_ > _AltDriver logging_

~~❗ There is one subsection left to be updated with the robot tab, in _Logging_ > _AltDriver logging_ where I think should be handled by a developer of the team~~  Update: solved

### CODE REVIEW

File: Docs/source/pages/advanced-usage.md [hunk 1]
```diff
@@ -314,6 +347,34 @@ In the routing table, the personal hotspot network would be secondary, therefore
...
+
+            .. tab:: Android
+
+                Use the following static methods from the **AltReversePortForwarding** class in your test file:
+
+                    - **Reverse Port Forwarding Android**
+                        .. code-block:: robot
+
+                            Reverse Port Forwarding Android    device_port=13000    local_port=13000    device_id= 
+
```
Comment (rucindrea) [inline Docs/source/pages/advanced-usage.md:362]: You could use `device_id=your_device_id`

Comment (iulianatodoran) [reply Docs/source/pages/advanced-usage.md:362]: fixed

File: Docs/source/pages/advanced-usage.md [hunk 2]
```diff
@@ -628,8 +715,13 @@ Logging inside the instrumented app can be configured from the driver using the
...
 
-        altDriver.set_server_logging(AltLogger.File, AltLogLevel.Off);
-        altDriver.set_server_logging(AltLogger.Unity, AltLogLevel.Info);
+        altDriver.set_server_logging(AltLogger.File, AltLogLevel.Off)
+        altDriver.set_server_logging(AltLogger.Unity, AltLogLevel.Info)
+
+    .. code-tab:: robot
+
+        Set Server Logging    File    Off
+        Set Server Logging    Unity    Info
```
Comment (melinda-nagy) [inline Docs/source/pages/advanced-usage.md:725]: Can you please put "Off" and "Info" params one below the other (same as File and Unity)? :D

File: Docs/source/pages/get-started.md [hunk 3]
```diff
@@ -486,6 +486,76 @@ AltTester® package contains AltDriver class used to connect to the instrumented
...
+            1. The AltTester® Desktop app is running.
+            2. The instrumented build is running.
+            3. Configure the IP of the build in order to match the IP of the machine (PC) the Desktop app is running on.
+            4. Then press the restart button in the AltTester® dialog and wait until the `Connected to AltTester® Server…` message is displayed in the dialog.
+
+        **Test Setup:**
+            - You can use any IDE you want. As a suggestion, we prefer `Visual Studio Code <https://code.visualstudio.com/>`_  and `PyCharm <https://www.jetbrains.com/pycharm/download/>`_.
+
+        **Writing tests:**
+            - First you need to create a new project with th Robot class under it, which can store your tests
```
Comment (melinda-nagy) [inline Docs/source/pages/get-started.md:508]: There is a typo here -> "th". Maybe you can replace it with "... a Robot class under it".

File: Docs/source/pages/get-started.md [hunk 4]
```diff
@@ -486,6 +486,76 @@ AltTester® package contains AltDriver class used to connect to the instrumented
...
+                - Next create the ``Setup`` and ``Teardown`` suites for your tests
+                    - In the ``Setup`` suite you need to include the initialization of the AltDriver - make sure to also add the ip, host and app name as parameters if the connection data for the connected instrumented app is not the default one
+                    - In the ``Teardown`` suite the AltDriver needs to be stopped                
+            .. code-block:: robot
+
+                *** Settings***
+                Library    AltTesterLibrary
+                Suite Setup    Initialize Altdriver
+                Suite Teardown    Stop Altdriver 
+            - The ``Test Cases`` section stores all your tests who should have a specific and relevant name
```
Comment (melinda-nagy) [inline Docs/source/pages/get-started.md:520]: "...which should have" instead of "who should have"

File: Docs/source/pages/get-started.md [hunk 5]
```diff
@@ -486,6 +486,76 @@ AltTester® package contains AltDriver class used to connect to the instrumented
...
+            - Examples:
+            .. code-block:: console
+
+                robot my_first_test.robot
+
+            .. code-block:: console
+
+                robot path/to/my_tests/
+
+            - After running the test the Robot will generate 3 files - the ``report.html`` and ``log.html`` files can be opened in the browser and for every test in ``log.html`` there will be a status highlight, the keywords used, as well as a potential error that might have occured during the test
```
Comment (melinda-nagy) [inline Docs/source/pages/get-started.md:539]: Maybe I am missing something here, but which are the 3 files that we are talking about?

Comment (iulianatodoran) [reply Docs/source/pages/get-started.md:539]: `log.html`, `report.html`, `output.xml`

Also, I took this information from Sandy's demo video.

Comment (melinda-nagy) [reply Docs/source/pages/get-started.md:539]: Ok, but is the "output.xml mentioned here, in doc?"

Comment (iulianatodoran) [reply Docs/source/pages/get-started.md:539]: no, but I will mention it to avoid confusion

**Comment by melinda-nagy** [comment]
LGTM besides the observations that were fixed in the meantime.

**Review by gomboskriszta15 — APPROVED** [review]

**Pull request merged**